ui/desktop/src/components/settings_v2/extensions/index.ts

```diff
@@ -77,6 +78,11 @@ export async function activateExtension(
     // First add to the config system
     await addExtensionFn(nameToKey(name), config, true);
 
+    if (config.type != 'stdio') {
```

Comment:
The V1 implementation supported the other kinds as well so this is a reminder to myself to go in very quickly after this and refactor to support HTTP+SSE ones as well.